docs/tutorials/06_DataAssociation-MultiTargetTutorial.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+6 - Data Association - Multi-Target Tracking Tutorial
+=====================================================
+"""
+
+# %%
+# Tracking multiple targets through clutter
+# -----------------------------------------
+#
+# More often than not, the difficult part of state estimation concerns the ambiguous association of
+# predicted states with measurements. This happens whenever there is more that one target under
+# consideration, there are false alarms or clutter, targets can appear and disappear. That is to say
+# it happens everywhere.
+#
+# In this tutorial we introduce **global nearest neighbour** data association, which
+# attempts to find a globally-consistent collection of hypotheses such that some overall probability
```

Comment:
Shouldn't really have used 'probability' here. 'Score' or 'measure' would be better.